src/nora/components/NoraButton/NoraButton.module.scss

```diff
@@ -0,0 +1,17 @@
+.Button {
+  font-weight: 500;
+  font-size: 13px;
+  height: var(--Space-32);
+  line-height: var(--Space-32);
+  letter-spacing: 0.015em;
```

Comment:
I've found letter spacing css to be so arbitrary, I've pretty much given up on kerning and tracking on the web unless it's a display heading or something that needs to be curated.